### PR TITLE
Improved resize borders

### PR DESCRIPTION
**Pull request by chevtche**

### CODE REVIEW

**Comment by chevtche** [comment]
retest this please

**Comment by chevtche** [comment]
retest this please

File: dc/core/resources/BorderRectangle.qml [hunk 1]
```diff
@@ -7,35 +7,51 @@ Rectangle {
     property int borderSize: Style.borderWidth
     property int border: modelData
     state: "inactive"
+    color: "#00000000"
```
Comment (rdumusc) [inline dc/core/resources/BorderRectangle.qml]: If this Rectangle is always transparent, use an Item element instead

File: dc/core/resources/BorderRectangle.qml [hunk 2]
```diff
@@ -7,35 +7,51 @@ Rectangle {
     property int borderSize: Style.borderWidth
```
Comment (rdumusc) [inline dc/core/resources/BorderRectangle.qml]: still used?

**Comment by rdumusc** [comment]
please assign someone when you open a PR, otherwise we loose track of them :-)

File: dc/core/resources/BorderRectangle.qml (hunk 1 above)
Comment (chevtche) [inline dc/core/resources/BorderRectangle.qml]: Done

File: dc/core/resources/BorderRectangle.qml (hunk 2 above)
Comment (chevtche) [inline dc/core/resources/BorderRectangle.qml]: No, Done.

**Comment by rdumusc** [comment]
+1

**Comment by rdumusc** [comment]
please rebase this change so that we can merge it

**Comment by chevtche** [comment]
Rebased and ready to merge.

**Pull request merged**